doc/api/crypto.md

```diff
@@ -4732,7 +4732,8 @@ added: v15.6.0
     **Default:** `false`.
 * Returns: {string}
 
-Generates a random [RFC 4122][] Version 4 UUID.
+Generates a random [RFC 4122][] Version 4 UUID. The UUID is generated using a
+cryptographic pseudo-random number generator.
```

Comment:
Optional suggestion: I know we use _pseudo-random_ in our docs, but I'm about to open a pull request to change it to _pseudorandom_ to comply with our style guide. (We also use _pseudorandom_ already n a few places so this is for consistency as well.)

```suggestion
cryptographic pseudorandom number generator.
```